tools/AnalyzerCodeGenerator/IncrementalGenerator/IncrementalGenerator.cs

```diff
@@ -11,15 +11,15 @@ namespace AnalyzerCodeGenerator
 {
     public class IncrementalGenerator
     {
-        private static string _masterCsvFile;
+        private static string _mainCsvFile;
```

Comment:
I think these should be reverted? They aren't referring the branch names

Reply:
Done! Thanks.